call of overloaded ‘ReadWriteJ()’ is ambiguous

I am getting a compiler complaining with the message in title. The issue seem to come from two lines in table.hpp
```
 int *ReadWriteJ(bool on_dev = true) { return J.ReadWrite(on_dev); }
 int *ReadWriteJ() { return J.HostReadWrite(); }
```
I suspect "bool on_dev = true" is optional, calling ReadWriteJ() can be either of definitions?  Maybe 
the second one is supposed to be HostReadWriteJ?

FYI, here is full text of error message.
```
table_wrap.cxx: In function ‘PyObject* _wrap_Table_ReadWriteJ__SWIG_1(PyObject*, Py_ssize_t, PyObject**)’:
  table_wrap.cxx:6689:41: error: call of overloaded ‘ReadWriteJ()’ is ambiguous
   6689 |       result = (int *)(arg1)->ReadWriteJ();
        |                       ~~~~~~~~~~~~~~~~~~^~
  In file included from /home/shiraiwa/venvs/py313/src/PyMFEM/build/bdist.linux-x86_64/wheel/mfem/external/ser/include/mfem/general/sets.hpp:17,
                   from /home/shiraiwa/venvs/py313/src/PyMFEM/build/bdist.linux-x86_64/wheel/mfem/external/ser/include/mfem/mfem.hpp:22,
                   from /home/shiraiwa/venvs/py313/src/PyMFEM/build/bdist.linux-x86_64/wheel/mfem/external/ser/include/mfem.hpp:2,
                   from table_wrap.cxx:3623:
  /home/shiraiwa/venvs/py313/src/PyMFEM/build/bdist.linux-x86_64/wheel/mfem/external/ser/include/mfem/general/table.hpp:146:9: note: candidate: ‘int* mfem::Table::ReadWriteJ(bool)’
    146 |    int *ReadWriteJ(bool on_dev = true) { return J.ReadWrite(on_dev); }
        |         ^~~~~~~~~~
  /home/shiraiwa/venvs/py313/src/PyMFEM/build/bdist.linux-x86_64/wheel/mfem/external/ser/include/mfem/general/table.hpp:149:9: note: candidate: ‘int* mfem::Table::ReadWriteJ()’
    149 |    int *ReadWriteJ() { return J.HostReadWrite(); }
```

--- a/general/table.hpp
+++ b/general/table.hpp
@@ -146,7 +146,7 @@ public:
    int *ReadWriteJ(bool on_dev = true) { return J.ReadWrite(on_dev); }
    const int *HostReadJ() const { return J.HostRead(); }
    int *HostWriteJ() { return J.HostWrite(); }
-   int *ReadWriteJ() { return J.HostReadWrite(); }
+   int *HostReadWriteJ() { return J.HostReadWrite(); }
 
    /// Sort the column (TYPE II) indices in each row.
    void SortRows();

